apps/calendar/src/app/[locale]/(root)/navbar-calendar-header.tsx

```
@@ -1,69 +1,56 @@
-} from '@tuturuuu/ui/select';
-import dayjs from 'dayjs';
-import {
-  Calendar as CalendarIcon,
-  ChevronLeft,
-  ChevronRight,
-} from 'lucide-react';
+import { ChevronLeft, ChevronRight } from 'lucide-react';
 import { useTranslations } from 'next-intl';
 
-export default function NavbarCalendarHeader({ locale }: { locale: string }) {
+export default function NavbarCalendarHeader({}: { locale: string }) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)